pontoon/base/templates/base.html

```diff
@@ -36,9 +36,6 @@
 
       {% include "header.html" %}
 
-      {% block before %}
-      {% endblock %}
-
```

Comment:
Are you sure this is not needed? If not, please restore it.

Reply:
Yeah, sorry it was required. It was creating error to select the custom homepage on the settings.